testing/citest/spinnaker_testing/dcoscli_agent.py

```diff
@@ -0,0 +1,35 @@
+# Standard python modules.
```

Comment:
Looks like I gave you bad advice and packages are encapsulated. I guess that makes sense given the __init__.py and other metadata type files in a package directory. I'd still like to move it out. Maybe a spinnaker/testing/citest/citest_contrib directory and put dcos_testing/dcos_agent or whatever you called it under there? It would be imported as citest_contrib.dcos_testing because of the path, but eventually would like to move it into the google/citest repo if you dont mind. At least take it out of the spinnaker-specific packages.